packages/SketchMorph2-Core.package/MorphicMonet.class/class/open.st

```diff
@@ -1,3 +1,4 @@
 instance creation
 open
+	MorphicMonet initialize.
```

Comment:
Is it smart to do this every time a `MorphicMonet` instance is created?

Reply:
The method works as follows:
```smalltalk
registerOpenCommand: anArray

self unregisterOpenCommand: anArray first.
OpenMenuRegistry addLast: anArray
```
Therefore it unregisters the command if already present and checking if present on our own before calling this method would add unneccessary complexity imo (also the other packages using this command are not doing it either as I looked through some of them).